i18n/zh/docusaurus-plugin-content-docs-kruisegame/current/user-manuals/crd-field-description.md

```diff
@@ -18,7 +18,8 @@ type GameServerSetSpec struct {
 	ServiceName          string             `json:"serviceName,omitempty"`
 
     // 保留的游戏服序号，可选项。若指定了该序号，已经存在的游戏服将被删除；而未存在的游戏服，新建时将跳过、不创建该序号
-    ReserveGameServerIds []int              `json:"reserveGameServerIds,omitempty"`
+    // 允许指定的序号的范围, 例如 [1, 3, 4, 5] 可以表示为 [1, 3-5]
+    ReserveGameServerIds []intintstr.IntOrString `json:"reserveGameServerIds,omitempty"`
```

Comment:
The previous translation was quite good in my opinion. I suggest revising it into English.

Reply:
Sorry, I don't quite understand which part should be translated into English.🤔